project_stock/views/project_task_view.xml

```diff
@@ -113,7 +121,6 @@
                         name="move_ids"
                         context="{'tree_view_ref': 'project_stock.view_stock_move_raw_tree', 'form_view_ref':'stock.view_move_form', 'default_company_id': company_id, 'default_state': 'draft', 'default_raw_material_task_id': id}"
                         attrs="{'readonly': [('done_stock_moves', '=', True),('stock_moves_is_locked', '=', True)]}"
-                        groups="stock.group_stock_user"
```

Comment:
Why removing this group? It shouldn't be unprotected. In any case, just add another invisible occurrence, but it seems another field that requires this one should be protected instead.

Reply:
OK as well.